apps/api/src/app/widgets/usecases/initialize-session/initialize-session.usecase.ts

```diff
@@ -35,10 +35,8 @@ export class InitializeSession {
       throw new ApiException('Please provide a valid app identifier');
     }
 
-    const inAppIntegration = await this.getDecryptedIntegrations.execute(
-      GetDecryptedIntegrationsCommand.create({
-        findOne: true,
-        active: true,
+    const inAppIntegration = await this.selectIntegration.execute(
```

Comment:
use the new use-case to check if the `in-app` integration is created... in the future, we might allow having many in-app integrations, that's why we've also introduced it here...